examples/DynamicCantileverCase/dynamic_cantilever_phase_space.py

```diff
@@ -0,0 +1,84 @@
+import matplotlib.pyplot as plt
```

Comment:
Lets add a small discription as `__docs__` here describing what is this file about?

Reply:
0f374e4937dd5108073a940676fec1cf7fb4143c